pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -543,6 +543,67 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("create", "delete", "deletecollection", "get", "list", "patch", "update", "watch"),
+					Resources: sets.NewString("imagestreamimages", "imagestreamimports", "imagestreammappings", "imagestreams", "imagestreams/secrets", "imagestreamtags"),
+				},
+				{
+					Verbs:     sets.NewString("create", "delete", "deletecollection", "get", "list", "patch", "update", "watch"),
```

Comment:
> roles, but not clusterroles?

Correct, this for a project scoped role, so clusterroles would have no effect and cloud the role.



> most of these verbs don't apply to lsar/lrar types... @deads2k do you care?

Not really.  All the resources are related to policy things and this groups it nicely.